package.json

```diff
@@ -33,7 +33,6 @@
   "scripts": {
     "setup": "npm install && npm run validate -s",
     "validate": "kcd-scripts validate",
-    "prepare": "npm run build",
```

Comment:
just a reminder that removing this means install will happen differently, build will no longer run on npm install, so it'll need to be done before packaging and publishing to npm

Reply:
Yeah, I'm pretty sure this was just left over from before we used semantic-release. We now have an explicit built in the release pipeline.